README.md

```diff
@@ -1,97 +1,41 @@
-
-```
-dnvm upgrade
-```
-
-> DNVM has the concept of a stable and unstable feed. Stable defaults to NuGet.org while unstable defaults to our dev MyGet feed. So if you add `-u` or `-unstable` to any of the install or upgrade commands you will get our latest CI build of the DNX instead of the one last released on NuGet.
-
-DNVM works by manipulating your path. When you install a runtime it downloads it and adds the path to the dnx binary to your `PATH`. After doing upgrade you should be able to run `dnvm list` and see an active runtime in the list.
-
-You should also be able to run `dnx` and see the help text of the `dnx` command.
-
-## Running the samples
-
-1. Clone the ASP.NET Core Home repository: https://github.com/aspnet/home
-2. Change directory to the folder of the sample you want to run
-3. Run ```dnu restore``` to restore the packages required by that sample.
-4. You should see a bunch of output as all the dependencies of the app are downloaded from MyGet.
-5. Run the sample using the appropriate DNX command:
-    - For the console app run  `dnx run`.
-    - For the web apps run `dnx kestrel`.
-6. You should see the output of the console app or a message that says the site is now started.
-7. You can navigate to the web apps in a browser by navigating to `http://localhost:5004`
-
-# Documentation and Further Learning
-
-## [Community Standup](https://www.youtube.com/playlist?list=PL0M0zPgJ3HSftTAAHttA3JQU4vOjXFquF)
+- [Community Standup](http://live.asp.net)
 The community standup is held every week and streamed live to YouTube. You can view past standups in the linked playlist.
-
-If you have questions you can also jump online during the next standup and have them answered live.
-
-## [Wiki Documentation](https://github.com/aspnet/Home/wiki)
-We have some useful documentation on the wiki of this Repo. This wiki is a central spot for docs from any part of the stack.
-
-If you see errors, or want some extra content, then feel free to create an issue or send a pull request (see feedback section below).
-
-## [ASP.NET/vNext](https://www.asp.net/vnext)
-The vNext page on the ASP.NET site has links to some TechEd videos and articles with some good information about ASP.NET Core (formerly known as ASP.NET 5).
-
-## [Roadmap] (https://github.com/aspnet/Home/wiki/Roadmap)
+- [Roadmap](https://github.com/aspnet/Home/wiki/Roadmap)
 The schedule and milestone themes for ASP.NET Core.
 
+## Getting ASP.NET Core
+
+- Stable builds of ASP.NET Core should be obtained from http://dot.net.
+- If you want to use nightlies then you need to:
+    - Obtain the latest nightly CLI from http://github.com/dotnet/cli
+    - Add a NuGet.Config to your app with the following content:
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+        <packageSources>
+            <clear />
+            <add key="AspNetCI" value="https://www.myget.org/F/aspnetcirelease/api/v3/index.json" />

```

Comment:
shouoldn't this be aspnetrelease and not aspnetcirelease? My understanding is that aspnetcirelease contains untested packages...